recipe/run_test.sh

```diff
@@ -2,98 +2,98 @@
 
 set +x
 
-if [[ -n "$TERM" && "$TERM" != dumb ]]; then
-    txtund=$(tput sgr 0 1)          # underline
-    txtbld=$(tput bold)             # bold
-    bldred=${txtbld}$(tput setaf 1) # red
-    bldgre=${txtbld}$(tput setaf 2) # green
-    bldyel=${txtbld}$(tput setaf 3) # yellow
-    bldblu=${txtbld}$(tput setaf 4) # blue
-    txtcya=$(tput setaf 6)          # cyan
-    bldwht=${txtbld}$(tput setaf 7) # white
-    txtrst=$(tput sgr0)             # reset
-fi
-
-ERROR=0
-SUCCESS=0
-
-
-
-pidof() {
-    local pids=$(ps axc | awk "{if (\$5==\"$1\") print \$1}" | tr '\n' ' ')
-    if [[ -n $pids ]]; then
-        echo "$pids"
-    else
-        return 1;
+if [[ "${target_platform}" == linux-aarch64 ]]; then
+   # Skip testing on aarch64 because 'ps' is not found
+   exit 0
+else
```

Comment:
Since we're exit'ing above, we don't need the else, if I'm not mistaken.

Reply:
I want to skip testing on aarch64. Without else I guess the build process won't know what to do with other platforms. But I'm not sure

Reply:
When the platform is linux-aarch64, you exit the script. So whatever comes after it will not be executed (on aarch64), anyway. It's definitely not wrong, it's just an unnecessary indent.

Reply:
Yeah. I don't realize how conda-build check what platforms it will build on. I'll try without else and unnecessary indent if it works

Reply:
Thank you. I've learnt a new thing)